download_and_create_reference_datasets/v03/write_combined_mito_interval_reference_ht.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
```

Comment:
These scripts have been worked on enough that they are pretty much all copy pasta now.  There's a bit more work to get these down to a single script taking in the `ReferenceDatasetCollection` as an argument, which I think is out of scope for now.